packages/lab/stories/dialog/dialog.stories.tsx

```diff
@@ -166,14 +174,8 @@ const AlertDialogTemplate: ComponentStory<typeof Dialog> = ({
         // focus the ok instead of the cancel button
         initialFocus={1}
       >
-        <DialogTitle>
-          {status![0].toUpperCase()}
-          {status!.slice(1)}
-        </DialogTitle>
-        <DialogContent>
-          A Dialog is a window that opens over the application content, focusing
-          the user's attention on a particular task or piece of information.
-        </DialogContent>
+        <DialogTitle>{title}</DialogTitle>
```

Comment:
If `title` defaults to `undefined`, is there a case to not render `DialogTitle`?

Reply:
I didn't mean to default it to undefined (was trying to see if i can override the default props) - the default should be the args given for each story and there is always one
i think conditional rendering title and content would be a different story altogether showcasing how it looks when empty (like empty formfield with no control) which wouldnt be encouraged, so i wont add it (at least for now, but will note to Ivan, if there's use cases for no title/content)